servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/FlushStrategyOnServerTest.java

```diff
@@ -216,10 +238,7 @@ private void sendARequest() throws Exception {
                 headersFactory.newHeaders().set(TRANSFER_ENCODING, CHUNKED), DEFAULT_ALLOCATOR,
                 from(DEFAULT_ALLOCATOR.fromAscii("Hello"), headersFactory.newTrailers()), false,
                 headersFactory);
-        channel.writeInbound(req);
-        for (Object item : req.messageBody().toFuture().get()) {
-            channel.writeInbound(item);
-        }
+        client.request(req.toRequest().toFuture().get());
     }
 
     static class OutboundWriteEventsInterceptor extends ChannelOutboundHandlerAdapter {
```

Comment:
the elements added to this queue maybe reference counted. We should call `ReferenceCounted.release(..)` to avoid leaking memory.

Reply:
Changed it to pass-through